packages/terra-avatar/src/terra-dev-site/doc/avatar/SharedUser.4.doc.mdx

```diff
@@ -0,0 +1,44 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+import SharedUser from '../example/shared-user/SharedUser?dev-site-example';
+import SharedUserSize from '../example/shared-user/SharedUserSize?dev-site-example';
+import SharedUserColorVariants from '../example/shared-user/SharedUserColorVariants?dev-site-example';
+
+import SharedUserPropsTable from '../../../../src/variants/SharedUser?dev-site-props-table';
+
+<Badge />
+
+# Shared User
```

Comment:
It looks like we copied over the content from `docs/shared-user.md/.`Do we need to delete `'../../../../docs/shared-user.md';`?

Reply:
updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6